src/main/java/com/plotsquared/plothider/storage/palette/LegacyPalettedContainer.java

```diff
@@ -25,6 +25,12 @@
 
 import java.util.List;
 
+/**
+ * Legacy and non-padded/overflow palette container implementation.
+ * Used by Minecraft 1.15-.
```

Comment:
This plugin effectively doesn't run on 1.15 anymore, no need to add such a class then I guess?